.github/workflows/spotless.yml

```
@@ -23,6 +23,17 @@ jobs:
       - name: Grant execute permission to Gradle wrapper
         run: chmod +x ./gradlew
 
+      - name: Decode keystore
+        if: matrix.os == 'ubuntu-latest'
```

actionlint (1.7.9)

property "os" is not defined in object type {}

(expression)